### PR TITLE
refactor: move dhedge to new plugin infra

### PR DESCRIPTION
**Pull request by guanbinrui**
Part of #3066.

### CODE REVIEW

File: packages/maskbook/src/plugins/dHEDGE/Dashboard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+const dashboard: Plugin.Dashboard.Definition = {
+    ...base,
+    init(signal) {},
+    GlobalInjection: function Component() {
+        return (
+            <>
+                <InvestDialog />
+            </>
+        )
```
Comment (septs) [inline packages/maskbook/src/plugins/dHEDGE/Dashboard/index.tsx:13]:
```suggestion
        return <InvestDialog />
```

**Comment by septs** [comment]
plz clear build error

File: packages/maskbook/src/plugins/dHEDGE/base.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import type { Plugin } from '@masknet/plugin-infra'
+import { DHEDGE_PLUGIN_ID } from './constants'
+import { DHEDGEIcon } from '../../resources/DHEDGEIcon'
+
+export const base: Plugin.Shared.Definition = {
+    ID: DHEDGE_PLUGIN_ID,
+    icon: <DHEDGEIcon />,
+    name: { fallback: 'dHEDGE' },
+    description: { fallback: 'Decentralized hedge funds on Ethereum.' },
+    publisher: { name: { fallback: 'Mask Network' }, link: 'https://mask.io/' },
```
Comment (Jack-Works) [inline packages/maskbook/src/plugins/dHEDGE/base.tsx:10]: It's by a third party. https://github.com/DimensionDev/Maskbook/pull/3119

File: packages/maskbook/src/plugins/dHEDGE/Dashboard/index.tsx (hunk 1 above)
Comment (septs) [reply packages/maskbook/src/plugins/dHEDGE/Dashboard/index.tsx:13]: @Jack-Works why

**Review by septs — DISMISSED** [review]

**Review by Jack-Works — DISMISSED** [review]

**Review by Jack-Works — APPROVED** [review]

File: packages/maskbook/src/plugins/dHEDGE/base.tsx [hunk 3]
```diff
@@ -7,7 +7,7 @@ export const base: Plugin.Shared.Definition = {
     icon: <DHEDGEIcon />,
     name: { fallback: 'dHEDGE' },
     description: { fallback: 'Decentralized hedge funds on Ethereum.' },
-    publisher: { name: { fallback: 'Mask Network' }, link: 'https://mask.io/' },
+    publisher: { name: { fallback: 'iRhonin' }, link: 'https://github.com/iRhonin' },
```
Comment (septs) [inline packages/maskbook/src/plugins/dHEDGE/base.tsx:10]: @Jack-Works i think need add a field, reference into bounty (issue)

Comment (Jack-Works) [reply packages/maskbook/src/plugins/dHEDGE/base.tsx:10]: Why is that a necessary part of plugin information?

**Review by septs — APPROVED** [review]

**Pull request merged**